src/server/pfs/pfs.go

```diff
@@ -73,23 +74,23 @@ type ErrCommitNotFinished struct {
 }
 
 func (e ErrFileNotFound) Error() string {
-	return fmt.Sprintf("file %v not found in repo %v at commit %v", e.File.Path, e.File.Commit.Branch.Repo.Name, e.File.Commit.ID)
+	return fmt.Sprintf("file %v not found in repo %v at commit %v", e.File.Path, pretty.CompactPrintRepo(e.File.Commit.Branch.Repo), e.File.Commit.ID)
```

Comment:
I tried to use `CompactPrint*` for more user-facing messages, and the various db key functions for where we were essentially creating a key anyway. I would have just implemented `.String()` for `Repo` etc but the proto already has that. It seems like `.Format()` might be higher priority, so that could be an alternative.